source/fundamentals/typescript.txt

```diff
@@ -79,81 +84,119 @@ You can find a code snippet that shows how to specify a type for the ``FindCurso
 class in the
 :ref:`Find Multiple Documents Usage Example <node-driver-find-usage-example-code-snippet>`.
 
-Limitations
------------
+To view an example of a mutually recursive type, which is not supported by the
+preceding classes, see the :ref:`<node-driver-limitations-mutual-recursion>` section.
 
-.. _node-driver-typescript-limitations-dot-notation:
+.. _node-driver-limitations:
 
-The driver cannot infer the type of values with keys containing **dot
-notation**. Dot notation is a property access syntax for navigating BSON objects.
-Click on the tabs to see code snippets that highlight this behavior:
+Limitations For Driver Version {+version+}
+----------------------------------
 
-.. tabs::
+The following subsections describe the TypeScript specific limitations of
+version {+version+} of the {+driver-long+}. 
 
-  .. tab:: Dot Notation
-     :tabid: dot-notation
+Many limitations of the {+driver-short+} relate to **recursive types**.
```

Comment:
I think "Many limitations of the <driver>" isn't an accurate generalization. What are the other limitations of the driver as a whole? Why are most of them related to recursive types?

Reply:
Update to "Many" to "All" as the page only discusses limitations related to recursive types.